src/sentry/static/sentry/app/routes.jsx

```diff
@@ -744,13 +749,24 @@ function routes() {
       />
       <Route path="/organizations/new/" component={errorHandler(OrganizationCreate)} />
       <Route path="/onboarding/:orgId/" component={errorHandler(OrganizationContext)}>
+        {/* The current (old) version of the onboarding experience does not
+            route to anything here. So even though this is new, the route can
+            live where it will eventually live. */}
+        <Route
+          path=":step/"
+          componentPromise={() =>
+            import(/* webpackChunkName: "OnboardingWizardNew" */ './views/onboarding/wizardNew')
```

Comment:
Do we view Onboarding as something important enough that we want in the main bundle?

Reply:
IMO split is okay since after they go through it they’ll never need it again.